docs/development/CI/infrastructure.md

```diff
@@ -1,23 +1,186 @@
+    linkStyle 3 stroke:MediumOrchid,stroke-width:2px;
+
+    linkStyle 4 stroke:Brown,stroke-width:2px;
+    linkStyle 5 stroke:Brown,stroke-width:2px;
+
+    linkStyle 6 stroke:DarkCyan,stroke-width:2px;
+    linkStyle 7 stroke:DarkCyan,stroke-width:2px;
+    linkStyle 8 stroke:DarkCyan,stroke-width:2px;
+
+    linkStyle 9 stroke:CornFlowerBlue,stroke-width:2px;
+
+    linkStyle 10 stroke:Tomato,stroke-width:2px;
+    linkStyle 11 stroke:Tomato,stroke-width:2px;
+    linkStyle 12 stroke:Tomato,stroke-width:2px;
+    linkStyle 13 stroke:Tomato,stroke-width:2px;
+
+    linkStyle 14 stroke:ForestGreen,stroke-width:2px;
+    linkStyle 15 stroke:ForestGreen,stroke-width:2px;
+    linkStyle 16 stroke:ForestGreen,stroke-width:2px;
+    linkStyle 17 stroke:ForestGreen,stroke-width:2px;
+```
+
+<!--
+@startuml workflows
+
+left to right direction
+skinparam linetype ortho
+
+'!include <C4/C4_Component>
+
+card always [
+always
+]
+file src_eve #Plum;line:MediumOrchid;text:black [
+eve sources changed
+]
+file src_car #Gold;line:DarkGoldenRod;text:black [
+cartesian sources changed
+]
+file src_nxt #APPLICATION;line:CornFlowerBlue;text:black [
+next sources changed
+]
+file src_sto #Aquamarine;line:DarkCyan;text:black [
+storage sources changed
+]
+file src_cab #Gold;line:DarkGoldenRod;text:black [
+cartesian backend sources changed
+]
+file cfg_wfl #Coral;line:Tomato;text:black[
+workflows changed
+]
+file other #PaleGreen;line:ForestGreen;text:black [
+other files changed, excluding examples (the examples folder) and docs (.md and .rst files)
+]
+file pkg_cfg #PaleGreen;line:ForestGreen;text:black [
+package config files changed
+]
+
+agent eve #Plum;line:MediumOrchid;text:black [
+Test Eve
+]
+agent car #Gold;line:DarkGoldenRod;text:black [
+Test Cartesian (CPU)
+]
+agent nxt #APPLICATION;line:CornFlowerBlue;text:black [
+Test Next (CPU)
+]
+agent sto #Aquamarine;line:DarkCyan;text:black [
+Test Storage (CPU)
+]
+agent qua [
+Code Quality
+]
+
+always  -[hidden]r-> src_eve
+src_eve -[hidden]r-> src_car
+src_car -[hidden]r-> src_nxt
+src_nxt -[hidden]r-> src_sto
+src_sto -[hidden]r-> src_cab
+src_cab -[hidden]r-> cfg_wfl
+cfg_wfl -[hidden]r-> other
+
+always  -d-> qua
+
+src_eve -[#MediumOrchid,thickness=2]d-> car
+src_car -[#DarkGoldenRod,thickness=2]d-> car
+src_sto -[#DarkCyan,thickness=2]d-> car
+cfg_wfl -[#Tomato,thickness=2]d-> car
+other   -[#ForestGreen,thickness=2]d-> car
+
+src_eve -[#MediumOrchid,thickness=2]d-> eve
+cfg_wfl -[#Tomato,thickness=2]d-> eve
+pkg_cfg -[#ForestGreen,thickness=2]d-> eve
+
+src_eve -[#MediumOrchid,thickness=2]d-> nxt
+src_nxt -[#CornFlowerBlue,thickness=2]d-> nxt
+src_sto -[#DarkCyan,thickness=2]d-> nxt
+cfg_wfl -[#Tomato,thickness=2]d-> nxt
+other   -[#ForestGreen,thickness=2]d-> nxt
+
+src_sto -[#DarkCyan,thickness=2]d-> sto
+src_cab -[#DarkGoldenRod,thickness=2]d-> sto
+cfg_wfl -[#Tomato,thickness=2]d-> sto
+pkg_cfg -[#ForestGreen,thickness=2]d-> sto
+
+@enduml
+-->
+![](workflows.svg)
```

Comment:
I personally like more the mermaid version because is inlined here and doesn't really require installing extra tools, but it's just a personal opinion. In any case, we need to drop one of them, so pick up your favorite diagram and delete the other one.

Reply:
I will keep mermaid, after fixing the readability for dark mode github.